test.sh

```diff
@@ -39,6 +47,7 @@ set -ex
 go test -count=1 "$@" ./...
 
 # Configure linters in .golangci.yml
-go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
-./bin/golangci-lint run
-
+if ! command -v golangci-lint >/dev/null; then
```

Comment:
Not good enough, we need that specific version, which is why we go install the specific version. Older versions can fail on newer Go releases and vice versa.

Reply:
Oh, indeed `golangci-lint` can be moody!
I now check the output of `golangci-lint version`.

Reply:
No need for all this complexity. If the right version is already installed, go install is very fast.